src/main/groovy/org/mockito/release/internal/gradle/BaseJavaLibraryPlugin.java

```diff
@@ -90,5 +91,9 @@ public void execute(MavenPublication publication) {
 
         //so that we flesh out problems with maven publication during the build process
         project.getTasks().getByName("build").dependsOn("publishToMavenLocal");
+        // TODO 1 is this dependency correct? Result of fetchAllContributorsFromGitHub is needed in
+        // TODO PomCustomizer.customizePom(project, publication);
+        // TODO 2 Where define dependency to ContributorsPlugin?
```

Comment:
I suggest we do it this way:

1. Contributors are meaningful in the scope of the entire project. So it makes sense to just have one fetcher task for the entire project.
2. ContinuousDeliveryPlugin is intended to be applied in root project. So perhaps it can apply the contributors plugin? Then it could do something like:

```java
project.allprojects(new Action<Project>() {
            public void execute(final Project project) {
                //...
                project.getPlugins().withType(BaseJavaLibraryPlugin.class, new Action<JavaLibraryPlugin>() {
                    public void execute(JavaLibraryPlugin plugin) {
                        project.getTasks().getByName("dudeThatGeneratesPom").dependsOn(fetcherTask);
                    }
                });
            }
        });
```